src/plugins/recordTypes/valuation/forms/default.jsx

```diff
@@ -0,0 +1,52 @@
+const template = (configContext) => {
```

Comment:
Don't need this file. In 4.1, if you diff tenants/pahma/pages/ValuationcontrolTemplate.html with the same file in defaults, you'll see that the only customization we made was to remove the id generator from valuationcontrolRefNumber. In 5.0 that's done in fields.js, so there's nothing left to be customized in the form.